src/Components/v2/Lightbox/Lightbox.tsx

```diff
@@ -206,8 +206,7 @@ class LightboxComponent extends React.Component<LightboxProps, LightboxState> {
     if (element) {
       this.setState({
         element,
-        // FIXME: convert to import('openseadragon) once force supports it
-        promisedDragon: Promise.resolve(require("openseadragon")),
+        promisedDragon: import("openseadragon"),
```

Comment:
Because this is in `componentDidMount` this chunk will only be pulled in clientside.